src/pages/Confirmation/ConfirmationStyle.scss

```diff
@@ -0,0 +1,109 @@
+@import "../../pages/Index/index.scss";
+
+.LessonTypeDisplayIcon {
+  width: 250px;
+  height: 250px;
+  top: 24px;
+  background: #b1b0ff;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  border-radius: 15%;
+  border-color: #FFFFFF;
+  display: inline-block;
+  text-align: center;
+  font-size: 40px;
+  font-weight: bold;
+  font-family: Roboto;
+}
+
+.FlexContainer .EditButtonConfirmPage {
+  display: flex;
+  color: $assignments-font-color;
+  background-color: $assignments-background-color;
+  font-family: $assignments-roboto-font;
+  font-weight: bold;
+  border-radius: 1em;
+  text-align: center;
+  height: 2.5vw;
+  width: 6vw;
+  margin-top:40vh;
+  margin-left:2vw;
+  display: inline-block;
+}
+
+.FlexContainer .ConfirmButtonConfirmPage {
+  display: flex;
+  color: $assignments-font-color;
+  background-color: $assignments-background-color;
+  font-family: $assignments-roboto-font;
+  font-weight: bold;
+  border-radius: 1em;
+  text-align: center;
+  height: 2.5vw;
+  margin-top: 40vh;
+  margin-left: 56vw;
+  width: 6vw;
+  display: inline-block;
+}
+
+.FlexContainer .WordDisplayBar {
+  display: flex;
+  background-color: #DADADA;
+  display: inline-block;
+  text-align: center;
+  padding-top: 3vw;
+  padding-left: 3vw;
+  padding-right: 3vw;
+  padding-bottom: 3vw;
+  flex-grow: 1;
+  height: 100vh;
+}
+
+.FlexContainer {
+  display: flex;
+  flex-direction: row;
+  align-content: flex-start;
+}
+
+.FlexContainer .CardsDisplaySection {
+  display: flex;
+  background-color: #EEEEEE;
+  flex-wrap: wrap;
+  flex-grow: 4;
+  height: 100vh;
+}
+
+.FlexContainer .FlexColumn {
+  display: flex;
+  flex-direction: column;
+}
+
+.LessonCard {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+  overflow: scroll;
+  height: 15vw;
+  width: 12vw;
+  border-radius: 20px;
+  margin: 2vw;
+  margin-bottom: 0vw;
+  background-color: $assignments-background-color;
+  color: $assignments-font-color;
+  font-family: $assignments-roboto-font;
+  font-size: 1.2em;
+}
+
+.FlexContainer .LessonNameDisplayHeader {
+  display: flex;
+  background-color: #6A6868;
```

Comment:
what color is this and do we need a variable for it?

Reply:
uhhhh i pulled from figma we might need them later so I made it a variable now